app/components/Wallet/useWallet.tsx

```diff
@@ -22,27 +24,45 @@ export const useWallet = () => {
 			const response = await Wallet.request(getBalanceMethodName, null);
 			if (response.status === "success") {
 				setBalance(response.result.total);
+			} else {
+				toast({
+					title: "Balance",
+					description: "Failed to get the balance",
+					variant: "destructive",
+				});
```

Comment:
make sure this message doesn't auto-disappear in case user wants to copy it. It should be manually closed. Same for other error messages.

Reply:
Done.